templates/.github/workflows/ci.yaml

```diff
@@ -133,5 +133,7 @@ jobs:
             otp: 26
           - elixir: 1.17
             otp: 27
+          - elixir: 1.18
```

Comment:
Should we keep the last two versions?

Reply:
I'd probably keep 1.16 and drop 1.15 but that's just my feelings

Reply:
Updated